### PR TITLE
[#7] [UI] As a user, I can log in with a Github Access Token

### PR DESCRIPTION
**Pull request by llleyelll**
Resolves https://github.com/nimblehq/github-notifications-mac/issues/7

## What happened 👀

- Implement `Login` screen.
 
## Insight 📝

- Show `Login` screen when open the app.
- Implement `TextLabelWithHyperlink` with `UIViewRepresentable` as markdown in SwiftUI only available on Xcode 13 beta. I've tried combining `Text` and `Link` but it can't display as the same sentence also.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/45258998/132657381-2f2a83a1-1f9e-4da2-96dc-6642f104bc65.mov

### CODE REVIEW

File: Source/Extensions/Font+Application.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+//  Created by Chananchida F. on 9/8/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+extension Font {
+
+    static let headlineTitle: Font = { Font.system(size: 22, weight: .bold) }()
```
Comment (blyscuit) [inline Source/Extensions/Font+Application.swift:13]: I prefer if we use the `Dynamic Type Sizes` section in https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/ for font sizing and add modifier in declarative fashion `Text("Simple Swift Guide").font(.headline).bold().italic()`. Have to wait for other for this.

File: Source/Module/Login/LoginView.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    private let loginDescription: [TextContent] = [
+        .text("To generate a token, go to Github, "),
+        .link("personal access tokens", link: "https://github.com/settings/tokens"),
```
Comment (suho) [inline Source/Module/Login/LoginView.swift:15]: Screen Shot 2021-09-09 at 18 34 01

We have markdown support, but still beta 😭

Comment (suho) [reply Source/Module/Login/LoginView.swift:15]: https://developer.apple.com/documentation/foundation/attributedstring

File: Source/Extensions/Font+Application.swift (hunk 1 above)
Comment (suho) [reply Source/Extensions/Font+Application.swift:13]: Yes, we can use `Dynamic Type Sizes`

**Review by suho — APPROVED** [review]

File: Source/Views/ButtonStyle/RoundedCornersButtonStyle.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import SwiftUI
+
+struct RoundedCornersStyle: ButtonStyle {
+ 
+    var color: Color = Color.black
+    
+    func makeBody(configuration: Self.Configuration) -> some View {
+        configuration.label
+            .padding()
+            .frame(width: 155.0, height: 30.0, alignment: .center)
```
Comment (vnntsu) [inline Source/Views/ButtonStyle/RoundedCornersButtonStyle.swift:18]: I think we shouldn't have a fixed size inside this style.
btw, we're having the same idea, take a look at #51

Comment (llleyelll) [reply Source/Views/ButtonStyle/RoundedCornersButtonStyle.swift]: I updated the code. Please help check 🙇‍♀️

File: Source/Extensions/Font+Application.swift (hunk 1 above)
Comment (llleyelll) [reply Source/Extensions/Font+Application.swift]: @blyscuit I updated the code. Please help check 🙇‍♀️

**Review by blyscuit — APPROVED** [review]

**Review by vnntsu — APPROVED** [review]

**Pull request merged**